site/content/extend/plugins/security.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Security
+date: 2018-01-09T02:26:54-05:00
+subsection: Plugins (Beta)
+draft: true
+weight: 35
+---
+
+# Security
+
+Plugins are powerful. You should only install plugins that you've thoroughly reviewed as they have the potential to compromise the security of your installation.
+
+## Server Considerations
+
+Plugins have the ability to execute arbitrary code on your server. If they aren't properly isolated, they can do just about *anything*. For example, they could read your config file to get your database password, connect to your database, then exfiltrate sensitive user information.
+
+### Sandboxing
```

Comment:
Should we mention this if for MM server v4.7 and onward?

Reply:
Probably.